document-store/src/integrationTest/java/org/hypertrace/core/documentstore/postgres/PostgresDocStoreTest.java

```diff
@@ -0,0 +1,159 @@
+package org.hypertrace.core.documentstore.postgres;
```

Comment:
Just a note: For this test to work, we will need a similar task or extension via Postgres docker image in `build.gradle.kts`. we can do that at the end once everything is fine. For now, we can test by starting the server locally.

e.g in build.gradle.kts for mongo
```
tasks.register<DockerPullImage>("pullMongoImage") {
  image.set("mongo:4.2.0")
}
```